benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_torchbench_amp_freezing_inference.csv

```diff
@@ -106,7 +106,7 @@ doctr_det_predictor,pass,5
 
 
 
-doctr_reco_predictor,pass,4
+doctr_reco_predictor,fail_accuracy,4
```

Comment:
Why this model also fail the accuracy? I thought it was `lcnet_50`, isn't it?

Reply:
doctr_reco_predictor also in inline_inbuilt_nn_modules_models list, same issue as lcnet_50.

Reply:
Why lcnet_50 is not affected by this PR?

Reply:
lcnet_50 also is affected by this PR.